cv_bridge/CMakeLists.txt

```diff
@@ -18,18 +18,27 @@ if(NOT ANDROID)
     find_package(Boost REQUIRED python)
   else()
     find_package(Boost QUIET)
-    if(Boost_VERSION LESS 106500)
+    if(Boost_VERSION VERSION_LESS 1.65)
```

Comment:
Should we do `if(Boost_VERSION_STRING VERSION_LESS "1.67")`? For two reasons:
* This check currently doesn't match the CMake `FindBoost` documentation.
* `Boost_VERSION_STRING` will give us more consistent string on the version format.